### ISSUE TITLE
Properties window sent to behind

### ISSUE THREAD

**Issue by KeremArdicli** [files-community/Files#15856]
### Description

When i click on properties button, a new window appers and disappear immediately. Further investigation revaled that window is sent behind.

### Steps To Reproduce

Enable Columns view
open a folder
click on properties button

### Requirements

properties window should be kept in foreground

### Files Version

3.5.0.0

### Windows Version

10.0.19045.4651

### Log File

[debug.log](https://github.com/user-attachments/files/16358794/debug.log)

**Comment by 0x5bfa** [comment]
Thanks for the report.

- Is the last open folder is empty? If so, please check out if this issue is the same as #13697.
- When you try to bring it foreground, it will be immediately brought behind and never can bring to foreground?

**Comment by KeremArdicli** [comment]
- Yes it only happens when the folder is empty.
- It depends. If I click on the pane of the new empty folder, then yes it is as you say. If I click on the list where the folder is located then it opens in the foreground :)

**Comment by KeremArdicli** [comment]
Yeah I have revealed a new issue with this subject. Please see the video attached. 

- Once i right click, menu does not fully open and then disappear if i do not right click on the most recent pane. 
-  Properties window open in background but comes to foreground. It directly opens in the foreground if i use the button on the main window rather than right clcik menu.
- When I double click on a pane, it content disappers and the content of same folder does not load again unless I open another 
folder. 
- If a file or folder is not selected on the most recent folder, properties open the main folder properties.

If you have further questions please tell me how I can be of further help.

https://github.com/user-attachments/assets/93c4c023-1955-4f14-af10-c3e5be569caa

**Comment by Josh65-2201** [comment]
That is a sperate issue #15713 with the selected folder in column view.
